src/components/squareComponent/squareComponent.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+
+class Square extends Component {
+
+    constructor(props) {
+        super(props) // call super parent class constructor props
+        this.state = {
+            showMarker: true, // if true then marker will be shown  
+            xValue: null  
+        } 
+    }
+
+    toggleSquare = (e) => {
```

Comment:
Where is the test(s) for this method?

Reply:
Hey Andy, thanks for the great feedback, I'm confused on how to test state in React..should this be in an expect() toEqual()..?

Reply:
I haven't tested state in react in a while, but maybe this blog post might help? https://medium.com/@newyork.anthonyng/testing-react-components-state-b57bfc712b90

Reply:
I currently have the test to pass which will now render the board when the start button is clicked and will default to no board shown when it is not clicked. Should now be able to merge with master..hopefully